torch/csrc/dynamo/guards.cpp

```diff
@@ -321,8 +321,8 @@ static PyTypeObject TensorGuardsType = {
 static PyObject* check_type_id(PyObject* dummy, PyObject* args) {
   // faster `lambda obj, expected: id(type(obj)) == expected`
   PyObject* obj;
-  unsigned long expected;
-  if (!PyArg_ParseTuple(args, "Ok", &obj, &expected)) {
+  unsigned long long expected;
+  if (!PyArg_ParseTuple(args, "OK", &obj, &expected)) {
```

Comment:
Nice fix (cc: @jansel ) though dynamo isn't really useful on Windows.4 

But imo it should be part of separate PR, as code as it is written right now is syntactically correct (although semantically wrong, as object id is 64 bit value. Also, why not use `uint64_t` to make it platform agnostic )

Reply:
CPython document says unsigned  long long but not  uint64_t, although in practice it must be uint64_t on all major OS.